tests/unit/components/MainTitle.spec.js

```diff
@@ -0,0 +1,16 @@
+import { mount } from "@vue/test-utils";
+import MainTitle from '@/components/MainTitle.vue';
+
+describe("MainTitle.vue", () => {
+    it("render props when passed", () => {
+        const wrapper = mount(MainTitle, {
+            propsData: {
+                text: "Hello World!",
+                subText: "Something went wrong..."
+            }
+        });
+        expect(wrapper.props().text).toContain("Hello World!")
+        expect(wrapper.props().subText).toContain("Something went wrong...")
+
+    })
```

Comment:
this test is just checking input: it check if props contains given text and tbh it does not make much sense, as two lines above you are passing those props 😉 

you should test component output so e.g. you can test if components actually renders those values. You need to find proper html elements in wrapper (`wrapper.find(...)`) and chech their `text()` if this matches values you passed.